packages/frontend/src/components/VariablesList/index.tsx

```diff
@@ -8,10 +8,18 @@ import ListItemText from '@mui/material/ListItemText'
 import { Variable } from 'helpers/variables'
 
 function makeListItemComponent(
+  variable: Variable,
```

Comment:
**Nit:** Ah when I discussed this I was thinking more along the lines of making this a standalone component, since in all cases the other props remain the same.

But this also works, doesn't matter.

```
function ListItemComponent({variable, onClick}:{
  variable: Variable,
  onClick?: (variable: Variable) => void
}): JSX.Element {
  const handleOnClick = useCallback(() => onClick && onClick(variable), [variable, onClick])
  if (onClick) {
    return <ListItemButton
      sx={{ pl: 4 }}
      divider
      data-test="power-input-suggestion-item"
      onClick={handleOnClick}
    />
  }

  return <ListItem
    sx={{ pl: 4 }}
    divider
    data-test="power-input-suggestion-item"
  />
}
```

Reply:
okayy I see, gets gets thanks but ill just keep the original code for now